images/gnome-mime-application-x-audacity-legacy-project.xpm

```diff
@@ -0,0 +1,527 @@
+/* XPM */
+static char * const gnome_mime_application_x_audacity_legacy_project_xpm[] = {
+"48 48 476 2",
```

Comment:
Did you find this somewhere or create it yourself? Why not use a more modern image format like SVG?